qpmodel/stmt.cs

```diff
@@ -782,13 +783,15 @@ Expr RepalceSuquerySelection(Expr e)
             return newselection;
         }
 
+
         internal void ResolveOrdinals()
         {
+            bool isToppestNode = true;
```

Comment:
Won't this be true for every recursive call to resolve sub-query ordinals? Is that the intent? On the otherhand, if only the outer most query should have this to set to true, then context->parent_ == null would work, doesn't it?